VideoAPI/VideoApi.Domain/RoomParticipant.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using VideoApi.Domain.Ddd;
+
+namespace VideoApi.Domain
+{
+    public class RoomParticipant : Entity<long>
+    {
+        public RoomParticipant(long roomId, Guid participantId)
+        {
+            RoomId = roomId;
+            ParticipantId = participantId;
+            EnterTime = DateTime.UtcNow;
```

Comment:
After the discussion, I believe we agreed `EnterTime` is no longer needed

Reply:
I am not sure, we agreed on LeaveTime? I specially asked confirmation on slack